lib/DBIx/Class/Manual/Joining.pod

```diff
@@ -14,16 +14,15 @@ If you don't know the basics of DBIx::Class, read
 L<DBIx::Class::Manual::Intro> first. Skip this section if you know how
 SQL joins work.
 
-But I'll explain anyway. Assuming you have created your database in a
-more or less sensible way, you will end up with several tables that
-contain C<related> information. For example, you may have a table
-containing information about C<CD>s, containing the CD title and its
-year of publication, and another table containing all the C<Track>s
-for the CDs, one track per row.
+If you have created your database sensibly, you will have several tables
+that contain C<related> information. For example, you may have a table
+representing C<CD>s, containing each CD's title and its year of
+publication, and another table representing all the C<Track>s for the
+CDs, one track per row.
 
-When you wish to extract information about a particular CD and all
-its tracks, You can either fetch the CD row, then make another query
-to fetch the tracks, or you can use a join. Compare:
+To retrieve information about a particular CD and all its tracks, either
+fetch the CD row, then make another query to fetch the tracks, or use a
+join. Compare:
```

Comment:
On Mon, Apr 11, 2016 at 03:29:08PM -0700, Peter Rabbitson wrote:
> I think this needs to be expanded a little - as `join` on its own
> won't fetch anything extra. Perhaps `L<join and
> select|DBIx::Class::Resultset/prefetch>` the extra data.

Good point.  I'll work on that.

> Also towards the end of this section there is a link to a reference
> manual on SQL-side join syntax, not sure if this is the best
> resource... Thoughts?

https://en.wikipedia.org/wiki/Join_%28SQL%29 provides a more database
agnostic reference and
https://blog.codinghorror.com/a-visual-explanation-of-sql-joins/
provides a popular visual summary.  How about I link to those instead?

Tom